manual/integration_with_other_tools.md

```diff
@@ -15,7 +15,9 @@ The [vim-rubocop](https://github.com/ngmy/vim-rubocop) plugin runs
 RuboCop and displays the results in Vim.
 
 There's also a RuboCop checker in
-[syntastic](https://github.com/scrooloose/syntastic).
+[syntastic](https://github.com/scrooloose/syntastic);
```

Comment:
The updated sentence doesn't read good to me. Is there some difference between syntanstic and the other tools?